index.html

```diff
@@ -77,6 +77,18 @@
           authors: [
             "Andrew Banks", "Ed Briggs", "Ken Borgendale", "Rahul Gupta"
         ]
+        },
+        "LWM2M":{
+          href: "https://www.omaspecworks.org/what-is-oma-specworks/iot/lightweight-m2m-lwm2m/",
+          title: "Lightweight M2M",
+          status: "",
+          publisher:"Open Mobility Alliance"
+        },
+        "OCF":{
+          href: "https://openconnectivity.org/specs/OCF_Core_Specification_v1.3.0.pdf",
+          title: "OCF Core Specification 1.3",
```

Comment:
Bindings may need the whole spec suite, so it would be better to quote this link:

https://openconnectivity.org/developer/specifications/